docs/source/release_notes/release-notes.rst

```diff
@@ -63,6 +85,8 @@ develop
            Previously, clients would immediately retry the connection on the node with a 503 two times (for a total of three attempts) before failing over.
```

Comment:
Not super clear to me what AtlasDB HTTP clients are actually used for. I'd clarify that this "manifests as Timelock clients failing over and retrying non-leaders unnecessarily...".